# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/config_override.h
# Commit: 74fa25299c53
# Full Hash: 74fa25299c53a238c11fa559f0efc7038518b5c2
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 16:14:38
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r cff6a02ad4ee -r 74fa25299c53 media/ffvpx/config_override.h
--- a/media/ffvpx/config_override.h	Wed Feb 28 12:50:26 2024 +0000
+++ b/media/ffvpx/config_override.h	Wed Feb 28 12:50:26 2024 +0000
@@ -1,11 +1,9 @@
 // This file contains overrides for config.h, that can be platform-specific.
 
-#ifdef MOZ_LIBAV_FFT
 #undef CONFIG_FFT
 #undef CONFIG_RDFT
 #define CONFIG_FFT 1
 #define CONFIG_RDFT 1
-#endif
 
 // override '#define EXTERN_ASM _' in config_generic.h to allow building with
 // gcc on arm